### ISSUE TITLE
Goals for the next OMO workshop

### ISSUE THREAD

**Issue by matentzn** [information-artifact-ontology/ontology-metadata#83]
@zhengj2007 and I are planning an OMO workshop for next year (March time), and currently thinking about the most important items to go over. It is a major problem that there are _many_ important issues to solve, and all of these issues will demand space during the meeting. As far as I understand, it is @zhengj2007 key concern to discuss the OMO scope: Which APs should be in, which should be out. For the ones that are in, are they required or optional?

But before we do that, we should agree on the philosophy of OMO, which is not exactly clear. I guess there could be two ways to define what OMO is conceptually:

1. The RO for Annotation properties. This means:
    - Anything goes: if someone wants a property "editors note", they can register it if it makes some basic sense (i.e. pertain to OBO ontology metadata).
    - An acceptance to OMO does _not_ imply OBO Foundry recommends using it.
    - Similar to RO, we could maintain an `omo-core.owl` which contains only the recommended and required properties and their value sets.
    - OBO ontologies may choose to import an OMO module instead of the whole thing, or core.
2. A small core vocabulary for APs on which we want to reconcile across OBO foundry ontologies.
    - If someone wants a new property "editors note", they add it wherever. If they want it to be added as a recommended property, we have some SOP to get it in.
    - OBO ontologies typically import the whole thing.

My assumption is most people are thinking option 1, plus perhaps a "omo-core" module.

For the workshop, I would suggest, rather than "rejecting" APs from OMO, which could be pretty time-consuming, focussing on what goes in omo-core, and for that, I would suggest to focus on the following main issues:

- Attribution: How do we unify contributor annotations across the board. Goal: Being able to aggregate contributions across OBO ontologies.
- Synonymy: How do we do synonyms (exact, related, close)? How does this fit with synonym types? (#70)
- Obsoletion model: How do we model obsoletions, replacements, obsoletion reasons etc.
- Term provenance: How do we track which term comes from where? (iao:importedFrom vs rdfs:isDefinedBy, etc). What about versions?
- Minimal term metadata model: what metadata _SHOULD_ be present on new terms?

I would encourage everyone to look through the tracker and label all priority issues with [workshop](https://github.com/information-artifact-ontology/ontology-metadata/labels/workshop). Looking forward to this!

**Comment by alanruttenberg** [comment]
@nico I agree to your enumerated points. I would add one, which is that APs
that are added should serve more than one ontology and be relatively
universal. And obviously, these APs should be minimized and efforts made to
not have relatively insignificant variants.
 I have the same objection to #1 as I raised about RO - violates the
well-scoped ontology rule, at least, and eventually yields a huge artifact
that is unpleasant to use. There's a place for smaller sets of annotation
properties that are tailored for use in a small set of domains.

**Comment by matentzn** [comment]
@alanruttenberg we are in full agreement on that.. Devil will be in the details of which APs we would like to be "relatively universal" :)

**Comment by matentzn** [comment]
Give @balhoff the opportunity to talk a few minutes about his experience with developing an OBO metadata shape with SHACL and SHEX
